docs/development.md

```diff
@@ -44,6 +44,17 @@ Cleartext is never stored in anything other than memory (and event then it will
 When we are working with a "password", it should always be a [SHA-256](https://en.wikipedia.org/wiki/SHA-2) hash of the cleartext.
 Any variables/arguments actually using cleartext should make that apparent with the variable name (i.e. the name should have "cleartext" in it).
 
+### Test Users
+
+All test users that are server-role@test.com are used to test the corresponding ServerRoles.
+These users are not enrolled in any test courses by default.
+
+The role@test.com users denote the corresponding CourseRole.
+To test courses actions, these users are enrolled in various test courses.
+All of these users are given ServerRoleUser to simplify test cases.
+The exception to this rule is owner@test.com, who is also a ServerRoleOwner.
```

Comment:
What about tests that need a course owner, but not a server owner?
You should probably just add another user instead.

Reply:
I will add users with a "course-" prefix for each role in course roles. All of these test users will be server users. I will convert the original (non-prefixed) test users (i.e. admin@test.com) to be varying server roles, like they were before. This will take too long to create and fix test cases, so it will accompany my next PR that expands test cases.

Reply:
I did not ask for that and that change is unrelated to this PR.
This documentation is already unrelated to your PR, what you are proposing here is to add more to your PR based on documentation that shouldn't be in your PR.